packages/extension/src/ui/features/accountNfts/useNfts.ts

```diff
@@ -1,24 +1,28 @@
 import useSWR from "swr"
-
+import { Network } from "../../../shared/network"
 import { BaseWalletAccount } from "../../../shared/wallet.model"
 import { getAccountIdentifier } from "../../../shared/wallet.service"
-import { isEqualAddress } from "../../services/addresses"
 import { SWRConfigCommon } from "../../services/swr"
-import { AspectNft } from "./aspect.model"
-
-interface IGetNft {
-  nfts?: AspectNft[]
-  contractAddress: string
-  tokenId: string
-}
+import { getNFT } from "./alephium-nft.service"
+import { laggy } from "./laggy"
 
-export const getNft = ({ nfts, contractAddress, tokenId }: IGetNft) => {
-  if (!nfts) {
-    return
-  }
-  const nft = nfts.find(
-    ({ contract_address, token_id }) =>
-      isEqualAddress(contract_address, contractAddress) && token_id === tokenId,
+export const useNFT = (
+  network: Network,
+  collectionId?: string,
+  nftId?: string,
+  account?: BaseWalletAccount,
+  config?: SWRConfigCommon,
+) => {
+  const { data: nft, ...rest } = useSWR(
+    account && collectionId && nftId && [getAccountIdentifier(account), collectionId, nftId, 'nft'],
+    () => (collectionId && nftId) ? getNFT(collectionId, nftId, network) : undefined,
+    {
+      revalidateOnFocus: false,
+      revalidateOnMount: false,
+      use: [laggy],
+      suspense: true,
+      ...config,
```

Comment:
The `useSWR` [documentation](https://swr.vercel.app/docs/revalidation) describes when auto revalidation occurs. Here I removed the `refreshInterval` and set `revalidateOnFocus` and `revalidateOnMount` to false. The reason is that NFT metadata rarely changes, and we refresh it when users click the NFT at [here](https://github.com/alephium/extension-wallet/pull/159/files#diff-65514291c3def75d3087efce0068f2dfef666124bbc047aa5caf428ad4b94190R94).

Reply:
We need to put `...config` in the first place, so that it will not be override.
```
{
  ...config,
  revalidateOnFocus: false, ...
}
```